### PR TITLE
feat: Portuguese translation

### PR DESCRIPTION
**Pull request by dsoaress**
Feat: Portuguese translation

### CODE REVIEW

**Comment by marierigal** [comment]
There are 2 missing translations if you want to had them to your commit 😉 
https://github.com/netlify/netlify-identity-widget/pull/353/files#diff-571ce32a17d6aa22fdd0942224a8ff44

**Comment by dsoaress** [comment]
> There are 2 missing translations if you want to had them to your commit 😉
> https://github.com/netlify/netlify-identity-widget/pull/353/files#diff-571ce32a17d6aa22fdd0942224a8ff44

Done 😊

File: src/translations/pt.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "forgot_password": "Esqueceu a senha?",
+  "recover_password": "Recuperar senha",
+  "send_recovery_email": "Enviar email de recuperação de senha",
+  "sending_recovery_email": "Enviando email de recuperação de senha",
+  "never_mind": "Deixa pra lá",
+  "update_password": "Atualizar senha",
+  "updating_password": "Atualizando senha",
+  "complete_your_signup": "Complete seu registro",
+  "site_url_title": "Configurações de desenvolvimento",
+  "site_url_link_text": "Limpar URL local",
```
Comment (amarilisd) [inline src/translations/pt.json:19]:
```suggestion
  "site_url_link_text": "Limpar URL do localhost",
```
Thanks! I'm wondering whether this line should refer to the `localhost` explicitly?

Comment (dsoaress) [reply src/translations/pt.json:19]: Thank you. Your suggestion works better.

**Pull request merged**